### PR TITLE
Adding session

### PR DESCRIPTION
**Pull request by kumarrakesh**
1. removing store, using session instead

### CODE REVIEW

File: routes/productsupload.js [hunk 1]
```diff
@@ -4,35 +4,46 @@ var bodyParser = require('body-parser');
...
+                if(err){
+                    throw new Error(err);
+                }
+
+                fileUploadStatus = productUploadHandler.uploadProduct();
+                console.log("Status ", fileUploadStatus);
+                if(fileUploadStatus == "SUCCESS"){
+                    console.log("store.get('user') ", store.get('user') );
+                    if(store.get('user') == "undefined") {
+                        throw new Error("Not Logged In");
```
Comment (mailamiton) [inline routes/productsupload.js]: pls remove store.

File: routes/index.js [hunk 2]
```diff
@@ -3,8 +3,9 @@ var router = express.Router();
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-console.log("index router is called");
-  res.render('login', {"auth":"true"});
+    console.log("index router is called");
+    req.session.userId;
```
Comment (mailamiton) [inline routes/index.js]: use of     req.session.userId ?

File: routes/productsupload.js (hunk 1 above)
Comment (mailamiton) [inline routes/productsupload.js]: pls remove store

File: routes/index.js (hunk 2 above)
Comment (kumarrakesh) [reply routes/index.js]: defining variables to be stored in session. in this case "userId"

**Pull request merged**